testutils/infra/device.py

```diff
@@ -126,6 +126,9 @@ def get_passive_partition(self):
     def get_reboot_detector(self, host_ip):
         return RebootDetector(self, host_ip)
 
+    def get_client_service_name(self):
+        output = self.run("if test -e /lib/systemd/system/mender.service; then echo mender; else echo mender-client; fi")
+        return output.stdout
```

Comment:
This should be wrong: `self.run` is already returning `.stdout` (this is a key difference between integration tests and the more modern suites like mender-binary-delta)

Reply:
Ah good catch. I didn't see that. Will fix!